packages/toolpad-components/src/Text.tsx

```diff
@@ -35,14 +34,28 @@ const gutters = (marginTop: number, marginBottom: number) => ({
 
 const MarkdownContainer = styled('div')(({ theme }) => ({
   display: 'block',
-  [`&:empty::before`]: { content: '""', display: 'inline-block' },
-  h1: { ...theme.typography.h1, ...gutters(16, 16) },
-  h2: { ...theme.typography.h2, ...gutters(12, 12) },
-  h3: { ...theme.typography.h3, ...gutters(12, 12) },
-  h4: { ...theme.typography.h4, ...gutters(12, 12) },
-  h5: { ...theme.typography.h5, ...gutters(4, 4) },
-  h6: { ...theme.typography.h6, ...gutters(4, 4) },
-  p: { margin: 0, marginBottom: 6 },
+  maxWidth: 'fill-available',
```

Comment:
```suggestion
```

This doesn't seem to do anything for me. Which situation is this olving?

Reply:
This is to introduce scrollbars in content that spans the width of the container:

Without `fill-available`:
<img width="966" alt="Screenshot 2023-02-16 at 5 19 12 PM" src="https://user-images.githubusercontent.com/19550456/219357220-626e5ffc-c70d-4e6c-8f0c-2402b014587f.png">

With:
<img width="917" alt="Screenshot 2023-02-16 at 5 19 29 PM" src="https://user-images.githubusercontent.com/19550456/219357288-40c6cf8d-f4fd-4944-92ff-633675bf010d.png">

Reply:
Oh, ok, I see. Does it have advantages over `maxWidth: '100%'`?

Reply:
Nothing specifically I guess, I can use `100%` as well